pkg/function/backup_data_using_kopia_server.go

```diff
@@ -223,24 +229,36 @@ func backupDataUsingKopiaServer(
 	return kopiacmd.ParseSnapshotCreateOutput(stdout, stderr)
 }
 
-func hostNameAndUserPassPhraseFromRepoServer(userCreds string) (string, string, error) {
+func hostNameAndUserPassPhraseFromRepoServer(userCreds, hostname string) (string, string, error) {
```

Comment:
```suggestion
func hostNameAndUserPassPhraseFromRepoServer(userCreds, hostName string) (string, string, error) {
```

should `hostName` be preferred?

Reply:
I see `hostname` is used as directly or as substring in other var name. I think we should change them to `hostName` or `HostName`. Whatever applicable.

Reply:
Yeah, seems good. Will make the changes !

Reply:
Done !